src/components/panel-body/home/component.jsx

```diff
@@ -1,29 +1,36 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect, useMemo } from 'react';
 
 import LateralMenu from '../lateralMenu';
 import Navigation from 'components/common/navigation/component';
 import PropTypes from 'prop-types';
 import { Route } from 'react-router-dom';
 import UEPage from 'components/panel-body/UEpage';
 import UESPage from 'components/panel-body/UESpage';
+import userIdbService from 'utils/indexeddb/services/user-idb-service';
 import { UserProvider } from './UserContext';
 import { version } from '../../../../package.json';
+import { DEFAULT_USER_DATA } from 'utils/constants';
 
 const Home = ({ match }) => {
   const [textSearch, setTextSearch] = useState('');
   const [openDrawer, setOpenDrawer] = useState(false);
-  // TODO : use indexedDB user info retrieved with synchronization
-  const user = {
-    civility: 'MISS',
-    firstName: 'Int',
-    lastName: 'Erviewer',
-    email: 'int.erviewer@mai.il',
-    phoneNumber: '0123456789',
-  };
+  const [user, setUser] = useState(DEFAULT_USER_DATA);
+
+  useEffect(async () => {
+    const getUser = async () => {
+      const idbUsers = await userIdbService.getAll();
+      const onlyUser = idbUsers?.[0];
+      return onlyUser;
+    };
+    const myUser = await getUser();
+    setUser(myUser);
+  }, []);
```

Comment:
take care with async function in useEffect. 
* setUser(myUser) can we done even if the component is not mounted anymore
* in StrictMode useEffect are called twice so be carefull to getUser only once.